test/catch2/live_restore/api/test_live_restore_fh_extent_import_export.cpp

```diff
@@ -197,10 +197,9 @@ __alloc_exent(WT_SESSION_IMPL *session, wt_off_t offset, size_t len)
 
 TEST_CASE("Live Restore extent export", "[live_restore],[live_restore_extent_import_export]")
 {
```

Comment:
Most changes to test code are me aligning the tests to our new assumptions of what folders can contain when WiredTiger starts up. For example the python changes are cleaning out the destination folder in between starting up WiredTiger